src/priorities/priorities_overlords.ts

```diff
@@ -22,7 +22,9 @@ export let OverlordPriority = {
 		destroy         : 300,
 		healPoint       : 301,
 		siege           : 302,
-		controllerAttack: 399
+		controllerAttack: 399,
+		harass			: 571,
```

Comment:
make this a separate category - maybe `outpostOffense` or something. priorities should be strictly monotonic by category